tests/e2e/baseapp/block_gas_test.go

```diff
@@ -175,7 +175,7 @@ func TestBaseApp_BlockGas(t *testing.T) {
 				require.Equal(t, []byte("ok"), okValue)
 			}
 			// check block gas is always consumed
-			baseGas := uint64(38012) // baseGas is the gas consumed before tx msg
+			baseGas := uint64(38142) // baseGas is the gas consumed before tx msg
```

Comment:
we should document the increase in gas for users.

Reply:
will add it in with the changelog

Reply:
added in the changelog